### PR TITLE
Iss207 -- Rename to share_poverty_exposure and share_poverty_exposure_quality

### LINKED ISSUE #207: 9. Update share of poor residents living in high-poverty neighborhoods
Rename to `share_poverty_exposure` and `share_poverty_exposure_quality`

### PR DESCRIPTION
**Pull request by malcalakovalski**
Resolves #207

### CODE REVIEW

**Review by wcurrangroome — CHANGES_REQUESTED** [review]
Whoops--updating the status of this review to "Request changes" rather than "Comment".

File: 06_neighborhoods/poverty-exposure/poverty-exposure-city-2021.qmd [hunk 1]
```diff
@@ -1030,18 +1030,18 @@ set_quality <- function(cv, exposure) {
...
-    poverty_exposure_black_quality = set_quality(cv = poverty_black_cv, exposure = poverty_exposure_black),
-    poverty_exposure_hispanic_quality = set_quality(cv = poverty_hispanic_cv, exposure = poverty_exposure_hispanic),
-    poverty_exposure_other_races_quality = set_quality(cv = poverty_other_races_cv, exposure = poverty_exposure_other_races),
-    poverty_exposure_white_nonhispanic_quality = set_quality(cv = poverty_white_nonhispanic_cv, exposure = poverty_exposure_white_nonhispanic)
+    share_poverty_exposure_quality = set_quality(cv = poverty_cv, exposure = poverty_exposure),
+    share_poverty_exposure_black_quality = set_quality(cv = poverty_black_cv, exposure = poverty_exposure_black),
+    share_poverty_exposure_hispanic_quality = set_quality(cv = poverty_hispanic_cv, exposure = poverty_exposure_hispanic),
+    share_poverty_exposure_other_races_quality = set_quality(cv = poverty_other_races_cv, exposure = poverty_exposure_other_races),
+    share_poverty_exposure_white_nonhispanic_quality = set_quality(cv = poverty_white_nonhispanic_cv, exposure = poverty_exposure_white_nonhispanic)
```
[the remark this answers is not in the document]
Comment (awunderground) [reply 06_neighborhoods/poverty-exposure/poverty-exposure-city-2021.qmd:1037]: Good catch. Updated.

**Review by wcurrangroome — CHANGES_REQUESTED** [review]
No changes needed.

**Review by wcurrangroome — APPROVED** [review]
This is good to merge.

**Pull request merged**